### ISSUE TITLE
Adding a difficulty tag.

### ISSUE THREAD

**Issue by anuva04** [ollelauribostrom/rebus#537]
## Expected Behavior

The rebuses should get progressively tougher as one keeps solving them (that is what a gamer would expect).

## Actual Behavior

The difficulty of successive rebuses is quite random.

## Steps to Reproduce the Problem

1. Add a new parameter "difficulty" as a number (maybe from 1-5) in each object of the rebuses.js array.
2. Sort the objects according to their difficulty and display them to the user from low to high difficulty.
3. Add a difficulty logo to each rebus card (maybe on top-right).

## Specifications

- Browser: Should work for all browsers.

**Comment by martinh-dev** [comment]
Hey @anuva04, i would like to work on that issue. Please assign it to me. Thanks.

**Comment by silverwavetidaldragon** [comment]
Yes, this would be so nice!  Also would work great with another one's progress bar one!
